modules/pel/peltool/peltool.py

```diff
@@ -713,68 +713,96 @@ def main():
     PELsPath = "/var/lib/phosphor-logging/extensions/pels/logs/"
     PELsArchivePath = "/var/lib/phosphor-logging/extensions/pels/logs/archive"
     inBMC = os.path.isdir(PELsPath)
-    parser = argparse.ArgumentParser(formatter_class=CustomFormatter, description="PELTools")
+    example_text = '''Examples:
+        -------------------------
+        List PELs based on the severity: peltool -l -S Predictive
+        List PELs with Only option: peltool -l -H -O
+        List PELs based on severity and Only option: peltool -l -O -S Predictive
+        Count PELs based on the severity: peltool -l -S Predictive -n
+        Count PELs with Only option: peltool -n -H -O
+        Count PELs based on severity and Only option: peltool -n -O -S Predictive
+        List PELs summary based on the severity: peltool -a -S Recovered
```

Comment:
I'd save the word 'list' for when it just does a list.  How about 'Display all PELs based on ' instead?